docs/Documentation/UserGuideV0.7.0/7-Tools-Sync.md

```diff
@@ -0,0 +1,278 @@
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>Boolean</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>false</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart receiver</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: IP_white_list</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>Set up a white list of sender IP addresses, which is expressed in the form of network segments and separated by commas between multiple network segments. When the sender transfers data to the receiver, only when the IP address of the sender is within the network segment set by the whitelist can the receiver allow the sync operation. If the whitelist is empty, the receiver does not allow any sender to sync data. The default receiver accepts all IP sync requests.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>String</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>0.0.0.0/0</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart receiver</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: update_historical_data_possibility</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>The processing strategy chosen by the sync receiver when merging the sync data.<br/>
+        1. If the sync data accounts for more than 50% of the update of the historical data (compared with the latest timestamp of the local storage group data),then it is recommended to select strategy. Setting the parameter to true, which has a greater impact on the write performance of the IoTDB system and occupies less CPU of the machine.<br/>
+        2. If the sync data accounts for less than 50% of the update of the historical data (compared with the latest timestamp of the local storage group data),then it is recommended to select strategy. Setting the parameter to false, which has little impact on the write performance of IoTDB system and takes up a large amount of CPU power.<br/>
+</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>Boolean</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>false</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart receiver</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: sync_server_port</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>Sync receiver port to listen. Make sure that the port is not a system reserved port and is not occupied. The parameter is_sync_enable is valid when it is set to true, and the parameter is_sync_enable is invalid when it is set to false.</td>
```

Comment:
> "The parameter is_sync_enable is valid when it is set to true, and the parameter is_sync_enable is invalid when it is set to false."

I think this sentence is somewhat confusing. Maybe you want to say: 

> This paramter is valid only when the parameter `is_sync_enable` is set to TRUE.

Reply:
fixed